### ISSUE TITLE
[roi_pool/roi_align] Support dropped for tuple of Tensors

### ISSUE THREAD

**Issue by frgfm** [pytorch/vision#2197]
## 🐛 Support of Sequence of Tensors was downgraded to list of Tensors

Hello there,

I used to have a `roi_align` applied to a Tuple of Tensors, which is not working anymore since I updated torchvision. I believe the change is due to this [commit](https://github.com/pytorch/vision/commit/505cd6957711af790211896d32b40291bea1bc21)

## To Reproduce

```
import torch
from torchvision.ops import roi_align

x = torch.rand((2, 32, 19, 19)) 
boxes = boxes = tuple([torch.tensor([[0, 0, 50, 50]], dtype=torch.float), torch.tensor([[50, 50, 100, 100]], dtype=torch.float)])
out = roi_align(x, list(boxes), (7, 7), spatial_scale=0.15)
```
which yields
```
AssertionError: boxes is expected to be a Tensor[L, 5] or a List[Tensor[K, 4]]
```

## Expected behavior

Since `torch.cat` [supports](https://pytorch.org/docs/stable/torch.html#torch.cat) all Sequence of Tensors, we might want to restore this support. I don't foresee any side effects or unwanted behaviours with this suggested change.

## Environment

 - PyTorch / torchvision Version: 1.5.0 / 0.6.0a0+82fd1c8
 - OS: Ubuntu 18.04.4 LTS 
 - How you installed PyTorch / torchvision: `conda`
 - Python version: 3.7
 - CUDA/cuDNN version: 10.2 (cuDNN 7.6.5)
 - GPU models and configuration: GeForce GTX 1050 (driver 440.82)

**Comment by fmassa** [comment]
Yes, this is a bug and it would be great to be fixed. Could you send a PR fixing it?
Should just be a matter of replacing
https://github.com/pytorch/vision/blob/eb84e5d5ec2e5477bcffd7196c52dc7a98bfd5eb/torchvision/ops/_utils.py#L30
with
```python
if isinstance(boxes, (list, tuple)):
```

**Comment by frgfm** [comment]
Yes, I intended to so, I was just checking it was not made on purpose in anticipation of future changes!
Here is the PR #2199 !

**Comment by frgfm** [comment]
Fixed by #2199

**Comment by fmassa** [comment]
Thanks for the fix @frgfm !
